Classes/Controller/ImageRenderingController.php

```diff
@@ -83,7 +83,7 @@ class ImageRenderingController extends \TYPO3\CMS\Frontend\Plugin\AbstractPlugin
      * @param array $conf TypoScript configuration
      * @return string HTML output
      */
-    public function renderImageAttributes($content = '', $conf)
+    public function renderImageAttributes($content, $conf)
```

Comment:
Instead of making the first parameter required, they should both be made optional - as they are literally not required.